src/renault_api/kamereon/helpers.py

```diff
@@ -17,8 +17,10 @@
 ]
 
 
-def update_schedule(schedule: models.ChargeSchedule, settings: dict[str, Any]) -> None:
-    """Update schedule."""
+def update_charge_schedule(
```

Comment:
Please keep a copy of `update_schedule` and mark it as deprecated, replaced by `update_charge_schedule` to avoid a breaking change.

Reply:
How do you want this ? Like [so](https://github.com/hacf-fr/renault-api/blob/4605f8ecf1386cdb5c30002fc3ccd2a219330e93/src/renault_api/renault_vehicle.py#L552) ?

Reply:
Yes, something like that...
```python
def update_schedule(schedule: models.ChargeSchedule, settings: dict[str, Any]) -> None:
        warn("This method is deprecated, please use update_charge_schedule.", DeprecationWarning, stacklevel=2)
        update_charge_schedule(schedule, settings)
```